### PR TITLE
doc: document increased redis memory requirements

### PR DESCRIPTION
**Pull request by ggilmore**

### CODE REVIEW

File: doc/admin/updates/kubernetes.md [hunk 1]
```diff
@@ -17,13 +17,6 @@ and any manual migration steps you must perform.
 
 ## 3.26 -> 3.27
 
-TODO
-
-*How smooth was this upgrade process for you? You can give us your feedback on this upgrade by filling
-out [this feedback form](https://share.hsforms.com/1aGeG7ALQQEGO6zyfauIiCA1n7ku?update_version=3.26).*
-
-## 3.26 -> 3.27
-
```
Comment (daxmc99) [inline doc/admin/updates/kubernetes.md:26]: Should we provide an update here as well?

**Review by daxmc99 — APPROVED** [review]

File: doc/admin/updates/kubernetes.md (hunk 1 above)
Comment (ggilmore) [reply doc/admin/updates/kubernetes.md:26]: We can't until the new release is cut.

Comment (ggilmore) [reply doc/admin/updates/kubernetes.md:28]: @daxmc99 is that alright?

Comment (daxmc99) [reply doc/admin/updates/kubernetes.md:28]: We can't provide an update here under ## unreleased ? Similiar to CHANGELOG?

**Pull request merged**